prow-configurations/config.yaml.tpl

```diff
@@ -36,6 +37,19 @@ presubmits: # runs on PRs
     spec:
       containers:
       - image: eu.gcr.io/kyma-project/snapshot/test/integration:0.0.1 # created by running `docker build -t <image> .` in the integration-job directory.
+            
+postsubmits:
+  {{ .OrganizationOrUser }}/kyma:
+  - name: kyma-integration
```

Comment:
IMO name and context should be kyma-integration-postsubmit or kyma-integration-master

Reply:
Done